spec/controllers/settings/applications_controller_spec.rb

```diff
@@ -139,14 +139,14 @@ def call_update
     context 'failure' do
       before do
         patch :update, params: {
-                id: app.id,
+          id: app.id,
                 doorkeeper_application: {
```

Comment:
Spacing off here as well

Reply:
I re-alligned the whole block -  I think this is better now.